apps/services/routers.py

```diff
@@ -257,4 +256,27 @@ async def read_request_details(
             user_request_id=user_request_id,
             session=session),
         "message": "Got request details"
-    }
\ No newline at end of file
+    }
+
+
+@services_router.post("/{university_id}/create_students/",
+                      name="create_students_from_file",
+                      response_model=JSENDOutSchema[Union[List[CreateStudentsOut], None]],
+                      summary="Create students from file",
+                      responses={200: {"description": "Successful create students from file response"}},
+                      tags=['Admin dashboard'])
+async def create_students_from_file(
+        request: Request,
+        university_id: int,
+        file: UploadFile = File(...),
```

Comment:
'...' sure should be shema like this? I'm not sure, but maybe it's not ok view

Reply:
@YehorVoitenko `...` Elipsis means that this attribute is required.

Reply:
Done

Reply:
resolved